### PR TITLE
read version properly

### PR DESCRIPTION
**Pull request by Meruja**

### CODE REVIEW

File: src/main/java/org/wso2/patchvalidator/util/Util.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 import java.sql.SQLException;
 import java.util.*;
```
Comment (thushaaanthan) [inline src/main/java/org/wso2/patchvalidator/util/Util.java]: Get only the needed library other than getting all from util. can you please change this?

Comment (Meruja) [reply src/main/java/org/wso2/patchvalidator/util/Util.java]: changed it in next PR

Comment (Meruja) [reply src/main/java/org/wso2/patchvalidator/util/Util.java]: made the changes in next PR

Comment (Meruja) [reply src/main/java/org/wso2/patchvalidator/util/Util.java]: fixed with dc52862

**Pull request merged**